src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,21 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+	
+	private Person person;
+	private Tag tag;
+	private char action;
+	
+	public Tagging(Person person, Tag tag, char action){
+		this.person = person;
+		this.tag = tag;
+		this.action = action;
+	}
```

Comment:
Please fix the indentation and change the tab to spaces in Eclipse, see [here](http://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs)